modules/@apostrophecms/schema/ui/apos/components/AposInputJoin.stories.js

```diff
@@ -0,0 +1,138 @@
+import {
+  withKnobs,
+  select,
+  optionsKnob as options,
+  boolean
+} from '@storybook/addon-knobs';
+
+import AposInputJoin from './AposInputJoin.vue';
+
+export default {
+  title: 'Inputs (Join)',
+  decorators: [ withKnobs ]
+};
+
+export const joinInput = () => {
+
+  const max = select(
+    'Limit', {
+      None: null,
+      One: 1,
+      Two: 2,
+      Three: 3
+    },
+    null
+  );
+
+  const prepoulate = select(
+    'Prepopulate', {
+      None: 0,
+      One: 1,
+      Two: 2,
+      Three: 3
+    },
+    null
+  );
+
+  const isDisabled = boolean('Is Disabled?', false);
+
+  return {
+    components: { AposInputJoin },
+    data () {
+      return {
+        field: {
+          name: 'myJoin',
+          label: 'Link to A Page',
+          help: 'Choose a page from the joiner to link to it',
+          type: 'join',
+          withType: 'product',
+          max
+        },
+        value: {
+          data: getData(prepoulate)
+        },
+        status: {
+          disabled: isDisabled
+        }
+      };
+    },
+    template: `
+      <AposInputJoin
+        :field="field"
+        :value="value"
+        :status="status"
+      />`
+  };
+};
+
+function getData(n) {
+  const data = [
+    {
+      '_id': 'htcuoykl9012j38ecrjghrn0c',
+      'published': true,
```

Comment:
We could clear out most of this mock data other than `title` and `_id`. Not critical, but nicer.